course_discovery/apps/api/serializers.py

```diff
@@ -423,7 +423,7 @@ def get_social_network_url(self, url_type, obj):
         return None
 
     def get_profile_image_url(self, obj):
-        return obj.get_profile_image_url
```

Comment:
how did this work before?

Reply:
`get_profile_image_url` function trying to get the image url from the `StdImageField` where we need the value of `profile_image_url`

https://github.com/edly-io/course-discovery/blob/burhan/EDLY-2836/course_discovery/apps/course_metadata/models.py#L619

Reply:
For backward compatibility can we do it like:
```python
def get_profile_image_url(self, obj):
    profile_image_url = obj.get_profile_image_url
    if not profile_image_url:
        obj.profile_image_url

return profile_image_url
```